### ISSUE TITLE
error message when trying to send tokens

### ISSUE THREAD

**Issue by midasteam** [MyEtherWallet/MyEtherWallet#1110]
- **I'm submitting a ...**

  - [ ] Feature request
  - [x] Bug report

- **Feature Request**

  - Briefly explain the feature you would like us to implement
  - What are the current issues this feature will solve
  - Is this feature cross compatible with other ethereum networks
  - Are you willing to help us with development support if required
  - Where can this be added? e.g: `/interface`, `/create-wallet`, etc.

- **Bug Report**
  - What is the current behavior?
  - If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem
  - What is the expected behavior?
  - What is the motivation / use case for changing the behavior?
  - Please tell us about your environment:
    - Operating system
    - Browser and version
  - Other information (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)
  - Where can this be found? e.g: `/interface`, `/create-wallet`, etc.

**Comment by midasteam** [comment]
I keep getting this error message when trying to send tokens:
'invalid argument 0: json: cannot unmarshal hex number with leading zero digits
into Go value of type *hexutil.Big'
Please tell me in plain english exactly what I need to change/do in order to rectify this
message
I have checked forums etc. but all they give is lots of code which I don't
understand (am non techie)

**Comment by SteveMieskoski** [comment]
@midasteam could you provide more information such as the specific token and what inputs you used.

**Comment by midasteam** [comment]
Hello
Wolf has sorted it for me
Thank you for your offer of assistance

Regards
Trevor Blake

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
